apps/envio/.npmrc

```diff
@@ -0,0 +1,16 @@
+# Copyright 2023-2024 LightDotSo.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
```

Comment:
Will remove, just copy pasta!

Reply:
Ah, mis-read, it is from your github pipeline, ok, no worries :D

Reply:
Haha sorry 🙌